components/AdminFunc/CreateClass.tsx

```diff
@@ -0,0 +1,189 @@
+            <option value="AVAILABLE">Available</option>
+            <option value="NOT AVAILABLE">Not available</option>
+          </select>
+        </label>
+        <label htmlFor="week">
+          Week of the Year
+          <input
+            type="number"
+            id="week"
+            name="week"
+            //the onchange works but display wrong
+            value={inputs.week}
+            onChange={handleChange}
+          />
+        </label>
+        <label htmlFor="day">
+          Day
+          <select id="day" name="day" onChange={handleChange} required>
+            <option value="">-- Pick one --</option>
+            <option value="Monday">Monday</option>
+            <option value="Tuesday">Tuesday</option>
+            <option value="Wednesday">Wednesday</option>
+            <option value="Thursday">Thursday</option>
+            <option value="Friday">Friday</option>
+            <option value="Saturday">Saturday</option>
+            <option value="Sunday">Sunday</option>
+          </select>
+        </label>
+        <label htmlFor="startTime">
+          Starts at:
+          <select
+            id="startTime"
+            name="startTime"
+            onChange={handleChange}
+            required
+          >
+            <option value="">-- Pick one --</option>
+            <option value="08:00">08:00</option>
+            <option value="09:00">09:00</option>
+            <option value="10:00">10:00</option>
+            <option value="11:00">11:00</option>
+            <option value="12:00">12:00</option>
+            <option value="13:00">13:00</option>
+            <option value="14:00">14:00</option>
+            <option value="15:00">15:00</option>
+            <option value="16:00">16:00</option>
+            <option value="17:00">17:00</option>
+            <option value="18:00">18:00</option>
+            <option value="19:00">19:00</option>
+            <option value="20:00">20:00</option>
+          </select>
+        </label>
+        {/* todo if there's time: add teacher schema and use map for options, add CRUD */}
+        <label htmlFor="teacher">
+          Teacher
+          <select id="teacher" name="teacher" onChange={handleChange} required>
+            <option value="">-- Pick one --</option>
+            <option value="Sam">Sam</option>
+            <option value="Michael">Michael</option>
+            <option value="Sophie">Sophie</option>
+            <option value="Mohamud">Mohamud</option>
+            <option value="Irene">Irene</option>
+            <option value="Guido">Guido</option>
+          </select>
+        </label>
+        <label>
+          Duration
+          <select
+            id="duration"
+            name="duration"
+            onChange={handleChange}
+            required
+          >
+            <option value="">-- Pick one --</option>
+            <option value="45 min">45 min</option>
+            <option value="60 min">60 min</option>
```

Comment:
The value don't have to have the word `min`. Doesn't matter; that is data that you need to process the form.
So wherever you process the form, you can add "min" if necessary, or don't and just use the amount of minutes.

Let me know if I'm not being clear